src/watcher.js

```diff
@@ -57,18 +57,22 @@ export default function onSourceChange(
     proxyFileChanges({artifactsDir, onChange, filePath, shouldWatchFile});
   });
 
-  log.debug(`Watching for file changes in ${watchFile || sourceDir}`);
+  log.debug(
+    `Watching ${watchFile ? watchFile.join(',') : sourceDir} for changes`
+  );
 
   const watchedDirs = [];
   const watchedFiles = [];
 
   if (watchFile) {
-    if (fs.existsSync(watchFile) && !fs.lstatSync(watchFile).isFile()) {
-      throw new UsageError('Invalid --watch-file value: ' +
-        `"${watchFile}" is not a file.`);
-    }
+    for (const path of watchFile) {
```

Comment:
@Dexterp37 would you mind to rename `const path` into `file` or `filePath` (or something similar and still short enough)? 

I think that it is pretty common in nodejs apps and scripts to use `path` as the name for the result of importing the 'path' nodejs module, and so I would prefer to avoid using it for the string of the file path.